packages/cw1155/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cw1155"
+version = "0.6.0-alpha1"
+authors = ["Huang Yi <huang@crypto.com>"]
+edition = "2018"
+description = "Definition and types for the CosmWasm-1155 interface"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/cosmwasm-plus"
+homepage = "https://cosmwasm.com"
+documentation = "https://docs.cosmwasm.com"
+
+[dependencies]
+cosmwasm-std = { version = "0.14.0-beta1" }
+schemars = "0.7"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+
+[dev-dependencies]
+cosmwasm-schema = { version = "0.14.0-alpha2" }
```

Comment:
Probably should be -beta1 as well for consistency. (Not that the schema package changed)

Reply:
done